pkg/scheduler/framework/v1alpha1/context_test.go

```diff
@@ -47,20 +47,20 @@ func TestPluginContextClone(t *testing.T) {
 	if err != nil {
 		t.Errorf("failed to read copied value: %v", err)
 	}
-	if v, ok := valueCopy.(*fakeData); ok && v.Data != data1 {
-		t.Errorf("clone failed, got %q, expected %q", v.Data, data1)
+	if v, ok := valueCopy.(*fakeData); ok && v.data != data1 {
+		t.Errorf("clone failed, got %q, expected %q", v.data, data1)
```

Comment:
nit: prefer "want" to "expected" https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures